documentation/release_notes.markdown

```diff
@@ -4,6 +4,73 @@ title: "Puppet Server: Release Notes"
 canonical: "/puppetserver/latest/release_notes.html"
 ---
 
+## Puppet Server 1.0.8
+
+In addition to several bug fixes, this release adds new HTTP client timeout settings, a special logfile to capture only HTTP traffic, and a JRuby tuning guide to help you get the best performance from Puppet Server.
+
+###New Features
+
+#### Added new http-client timeout settings
+
+We've exposed two new HTTP client timeout settings, for idle and connection timeouts.
+
+* [SERVER-449](https://tickets.puppetlabs.com/browse/SERVER-449) - Expose http-client timeouts from Puppet Server http_connect_timeout and http_read_timeout.
+
+####Enabled HTTP traffic logs 
+
+This version of Puppet Server has a special-purpose logfile to capture only the HTTP traffic. This should work out of the box, but you can [configure the location and the format](./configuration.markdown#http-traffic) of the logfile.
+
+* [SERVER-319](https://tickets.puppetlabs.com/browse/SERVER-319)
+
+#### Added Puppet Server JRuby tuning guide
+
+We've added a new [Tuning Guide](./tuning_guide.markdown) to help you improve your Puppet Server performance by tuning your number of JRubies and your JVM heap size.
+
+* [SERVER-379](https://tickets.puppetlabs.com/browse/SERVER-379) - Tuning guide for JRubies, Heap size, etc.
+
+###Bug Fixes
+
+####Fixed an issue where Puppet Server couldn't start after reboot
+
+Previously, Puppet Server failed to start after a reboot on some systems (notably RHEL 7 and Ubuntu 14.4). This was because the `/var/run/` directory, needed by Puppet Server, was being destroyed on reboot. This issue has been fixed.
+
+* [SERVER-404](https://tickets.puppetlabs.com/browse/SERVER-404) - Properly create /var/run/puppetserver dir in FOSS packaging
+
+####JRuby default borrow timeouts are significantly extended
+
+The default JRuby pool borrow timeout was extended from 60 seconds to 20 minutes. This allows enough time for realistic expensive catalog compilations while avoiding indefinite hanging.
```

Comment:
This wasn't really clear from the ticket description but actually in SERVER 1.0.2 and earlier, the JRuby pool borrow timeout was infinite/indefinite and was not user-configurable.  As of SERVER 1.0.8, however, there is a new setting in the puppetserver.conf file, under the `http-client` section, called `borrow-timeout`.  If the user hasn't specified a value for that setting, Puppet Server will use 20 minutes as a default.

We had actually done the first pass at this work on the Puppet Server master branch for early 2.0 development and committed a default of 60 seconds.  We later bumped that up to 20 minutes, though - before the first 2.0 RC was released.  We haven't done any releases of Puppet Server where there was a 60 second default, so probably not worth mentioning that here.

I updated the description in SERVER-408 to hopefully make this a little more clear.